final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +119,95 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+class Arch_Enemy
+  attr_accessor :name, :smell, :weight, :citiesDestroyed, :luckyNumbers, :address
+  class Address
+    attr_accessor :number, :street, :state, :zip
+  end
+
+  def initialize(name, smell, weight, citys, lucky, address)
+    @name = name
+    @smell = smell
+    @weight = weight
+    @citiesDestroyed = citys
+    @luckyNumbers = lucky
+    @address = Address.new()
+  end
+
+end
+
+scary_monster = Arch_Enemy.new("Bill", "Smells like lilacs", 165, ["Frankfurt", "Munich"], [7, 14, 21], ("21 Washington st, WA, 51204"))
+
+puts " "
+
+pp scary_monster
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
 
 # - Create 2 instances of your SuperHero class
-
-
+class Superhero
+  attr_accessor :name, :super_power, :age, :arch_nemesis, :power_level, :energy_level
+  def initialize(n, sp, age)
```

Comment:
I totally see why you used these shortened variables and I get it - but the person interviewing your code who could potentially hire you won't want to train you out of it, and your PRs on the job won't get approved - so I know it's a few more keystrokes but get out of that habit sooner than later!!